Automattic/woocommerce-payments
# Add KYC additional info modal and notice banner

Fixes #4849 

#### Changes proposed in this Pull Request





- Adds a new `Notice` which is displayed on the onboarding page, as shown in the designs referenced in #4849
- Adds a new modal which displays a help guide on why we ask for verification information during the onboarding process. This is triggered by clicking on the notice.





#### Testing instructions





* As a new account (You can use dev tools to Force onboarding and act as disconnected from WCPay), visit the `Payments` page.
* Verify the new notice is visible - check the WooCommerce Payments link works, and check that the copy reads well.
* Click the link to open the modal.
* Verify the modal opens. Double check the content to ensure it reads well with no typos.
* Verify the modal closes when either the `Got It` button is clicked, or the close icon is clicked.

-------------------


**Post merge**
